src/HealthCheck.Mvc/Models/HealthCheckResultModel.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using HealthCheck.Core;
+using HealthCheck.Core.Results;
+
+namespace HealthCheck.Mvc.Models
+{
+    public class HealthCheckResultModel
+    {
+        public string DateTimeString { get; }
+        public bool Passed { get; }
+        public string Output { get; }
+        public HealthCheckResultSummary Summary { get; }
+        public IEnumerable<IGrouping<string, ICheckResult>> Sections { get; }
+        public HealthCheckResultModel(HealthCheckResult result)
+        {
+            DateTimeString = DateTime.UtcNow.ToString("G");
+            Output = result.Output;
+            Passed = result.Passed;
+            Summary = new HealthCheckResultSummary
+            {
+                TotalChecks = result.Results.Length,
+                PassedChecks = result.Results.Count(r => r.Passed),
+                FailedChecks = result.Results.Count(r => !r.Passed),
+            };
+            Sections = result.Results
+                .Select(r =>
+                {
+                    if (string.IsNullOrEmpty(r.SectionName))
+                    {
+                        r.SectionName = "#";
```

Comment:
Why "#"?

Reply:
If you will look at mvc page of healthcheck you shall admit that having empty title is looking bad. $ looks good)

Reply:
Ok.